tests/io/lobster/test_inputs.py

```diff
@@ -770,8 +771,8 @@ def test_attributes(self):
         atomlist = ["O1", "Mn2"]
         types = ["O", "Mn"]
         num_atoms = 2
-        assert charge_Mulliken == self.charge2.Mulliken
-        assert charge_Loewdin == self.charge2.Loewdin
+        assert charge_Mulliken == self.charge2.mulliken
```

Comment:
Could you keep the old tests for now as well?

Reply:
Sure

Reply:
Done